velox/type/DecimalUtil.h

```diff
@@ -249,6 +249,189 @@ class DecimalUtil {
     }
   }
 
+  // Get the java method BigInterger#toByteArray() array length.
+  inline static int32_t getByteArrayLength(int128_t value) {
```

Comment:
Would you move the implementation to the .cpp file? Also, please, move private methods which do not need access to class member variables to free functions in the .cpp file.

Reply:
Fixed, can you help review again?